libr/core/cmd_search.c

```diff
@@ -1688,6 +1688,10 @@ static int emulateSyscallPrelude(RCore *core, ut64 at, ut64 curpc) {
 }
 #endif
 
+static inline bool cmd_search_isThumb(RCore *core) {
+	return (!strcmp (core->anal->cur->arch, "arm") && core->anal->bits == 16);
```

Comment:
Dupped. This function is already defined with another name above

Reply:
Different file, different name. I could see pulling them into a single helper include instead